contrib/compose/example-app/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM debian
```

Comment:
Couple points.

1) We don't need to build from debian. busybox, alpine, or even "SCRATCH" would do fine.
2) We can just build an example app and push it to quay.io under the coreos name. The dockerfile can then live under examples.
3) Why do you need dexctl here?